### PR TITLE
Many fixes

### PR DESCRIPTION
**Pull request by gazuntype**
- add simulator mat to remote raw image
- fix big white raw image when updating plugin
- stop showing simulator when in logging-in process. only show when logged out or logged in.
- plugin can’t upload filter to test env. upload button does nothing. fixed this
- fix bug of login always being expired in plugin

### CODE REVIEW

File: Editor/Authentication.cs [hunk 1]
```diff
@@ -56,7 +56,7 @@ public class Authentication {
         public string LoginToken { get { return loginData.idToken; } }
         public string Uid { get { return loginData.localId; } }
 
-        public bool IsLoginExpired { get { return _expiryTime > DateTime.Now; } }
+        public bool IsLoginExpired { get { return _expiryTime < DateTime.Now; } }
```
[the remark this answers is not in the document]
Comment (gazuntype) [reply Editor/Authentication.cs:59]: haha, yeah you did. 😂

File: Editor/Simulator/Simulator.cs [hunk 2]
```diff
@@ -94,6 +97,9 @@ public class Simulator : SimulatorBase {
 
     protected override void OnEnable() {
         base.OnEnable();
+        if (!EditorApplication.isPlaying) {
+            _remoteFeed.gameObject.SetActive(false);
```
Comment (chornesays) [inline Editor/Simulator/Simulator.cs:101]: someday.....twould be nice to get this going without having to hit play

File: Editor/Simulator/Simulator.prefab [hunk 3]
```diff
@@ -12,7 +12,7 @@ GameObject:
   - component: {fileID: 1871096965488121260}
   - component: {fileID: 7382256431116267783}
   m_Layer: 5
-  m_Name: RawImage
+  m_Name: RemoteFeed
```
Comment (chornesays) [inline Editor/Simulator/Simulator.prefab:15]: 👏

**Review by chornesays — APPROVED** [review]
merge when ready

File: Editor/Simulator/Simulator.cs (hunk 2 above)
Comment (gazuntype) [reply Editor/Simulator/Simulator.cs:101]: 👀

**Pull request merged**